phpstan.neon

```diff
@@ -1,5 +1,4 @@
 includes:
-    - /composer/vendor/phpstan/phpstan-webmozart-assert/extension.neon
```

Comment:
Why did you remove this extension? Is it not needed for the assertion support?

Reply:
arg, shouldn't have included that in the commit

Reply:
I remove it locally because the path is not valid, but I revert that before committing usually.

I'm a little surprised it didn't seem to cause issue with the build though...

I reverted it